public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx

```diff
@@ -208,6 +210,10 @@ export class ManagedIndices extends MDSEnabledComponent<ManagedIndicesProps, Man
     }
   }
 
+  componentWillUnmount() {
+    this._isMounted = false;
```

Comment:
this isn't wrong but. you can also have a return statement that in the componentDidMount that sets this to false as well

like 

```
componentDidMount() {
  this._isMounted = true;

  return () => {
    this._isMounted = false;
    // Perform any necessary cleanup tasks here
  };
}
```

Reply:
the return function should be invoking and executing what ever in its return statement